TodoParser.py

```diff
@@ -24,22 +25,30 @@ class TodoParser(object):
     ISSUE_URL_PATTERN = re.compile(r'(?<=Issue URL:\s).+', re.IGNORECASE)
     ISSUE_NUMBER_PATTERN = re.compile(r'/issues/(\d+)', re.IGNORECASE)
 
-    def __init__(self):
+    def __init__(self, options=dict()):
```

Comment:
All changes in this section (through new line number 51) have to do with allowing the `identifiers` to be specified as an argument to the `TodoParser` constructor. I found this a cleaner method of configuring `TodoParser` when using it in the unit tests. The old method of setting it via an environment variable is still supported, and still has the highest priority, so this is backwards-compatible.

As an aside, it may be helpful to eventually support all configurable items via this `options` dictionary, but that's a separate discussion.